packages/react/src/ActionList/ActionList.features.stories.tsx

```diff
@@ -73,7 +74,8 @@ export const WithVisualListHeading = () => (
       </ActionList.Item>
     </ActionList.Group>
 
-    <ActionList.Group title="Advanced">
+    <ActionList.Group>
+      <ActionList.GroupHeading as="h4">Advanced</ActionList.GroupHeading>
```

Comment:
Just checking, do we have a story that still uses the old API? Just to keep it around for regressions

Reply:
Yes! I added them as dev stories https://github.com/primer/react/blob/main/packages/react/src/ActionList/ActionList.dev.stories.tsx and playwright snapshots them too to catch any regression 😈